### DISCUSSION TITLE
[Phaser Plugin] New Skybox

### DISCUSSION THREAD

**Discussion by randall-s-lee** [enable3d/enable3d#166] [Help / Questions] [answered]
Good morning, or whenever it is because the internet is always...

I'm looking for help on how to add a new skybox to the scene. I'm working on a project that involves ships in space. I need to change the blue sky to a big space bubble. It's likely easy and I'm just dumb. I'm having a hard time finding it in the documentation. I look forward to hearing how easy it is. Thanks, everyone.

**Comment by yandeu** [comment, marked as answer]
To remove the default sky, you have to remove it from warpSpeed: `this.third.warpSpeed('-sky')`

Then, you can add your own. 

Unfortunately, I'm not a pro in this area. 

Please have a look at the [three.js forum](https://discourse.threejs.org/).

**Reply by randall-s-lee** [reply]
okay I'll try looking at what is out there.

**Reply by N8python** [reply]
I found this: https://github.com/jeromeetienne/threex.daynight

**Reply by randall-s-lee** [reply]
@N8python I figured out how to change the background for the camera. Now I'm working on making the objects glow and the ground reflective.

**Reply by N8python** [reply]
Awesome - good luck!
